gems/smithy-client/spec/smithy-client/waiters_spec.rb

```diff
@@ -0,0 +1,708 @@
+# frozen_string_literal: true
+
+require_relative '../spec_helper'
+
+module Smithy
+  module Client
+    module Waiters
```

Comment:
I just noticed this, but should not be under module Waiters if you are describing Waiters